packages/react-ui/storybook/stories/Introduction.stories.mdx

```diff
@@ -2,36 +2,43 @@ import { Meta, Description } from '@storybook/addon-docs/blocks';
 
 <Meta title="Introduction" />
 
-# CARTO for React - UI Components
+# CARTO for React - Widgets and UI Components
 This storybook includes the UI Components for CARTO for React (@carto/react-ui).
```

Comment:
`This storybook includes the UI Components for CARTO for React (@carto/react-ui) and the powerful Widgets (@carto/react-widgets).`  Adding something like that?

Reply:
LGTM! Little change..
`This storybook includes the UI Components (@carto/react-ui) and the powerful Widgets (@carto/react-widgets) for CARTO for React`
What you think?